jib-gradle-plugin/src/main/java/com/google/cloud/tools/jib/gradle/TaskCommon.java

```diff
@@ -43,6 +44,14 @@ static War getWarTask(Project project) {
     if (warPluginConvention == null) {
       return null;
     }
+
+    if (project.getPlugins().hasPlugin("org.springframework.boot")) {
+      Task bootWar = project.getTasks().findByName("bootWar");
+      if (bootWar != null) { // Spring Boot > 2.0
+        return (War) bootWar;
+      }
+    }
+
     return (War) warPluginConvention.getProject().getTasks().findByName("war");
```

Comment:
Question as I am not a Gradle expert: can we just to `project.getTasks().findByName("war")` similarly, or is getting `warPluginConvention` still necessary? And should we still check at the beginning `if (warPluginConvention == null)` even when the bootWar case?

Reply:
Yes. `project` and `warPluginConvention.getProject()` will evaluate to the very same object instance.

Instead of checking for the presence of the `WarPluginConvention` it would be simpler to just check for the presence of the `WarPlugin` itself. See #1788 and #1789 for more information about this.

Reply:
Sounds good. Then, could you update `getWarTask()` here that way? (Check `WarPlugin` at the beginning, and don't use `warPluginConvention.getProject()`, right?)

Reply:
Sure. I'll do that.

Reply:
Done